src/io/perun.clj

```diff
@@ -36,50 +36,48 @@
 
 (deftask markdown
   "Parse markdown files"
-  [d datafile        DATAFILE        str "Target datafile with all parsed meta information"
-   f create-filename CREATE_FILENAME str "Function that creates final target filename of the file"]
+  [f create-filename CREATE_FILENAME str "Function that creates final target filename of the file"]
```

Comment:
not really related to this PR but `create-filename` does not seem to be used anywhere?